src/assets/css/_fonts.scss

```diff
@@ -0,0 +1,8 @@
+@font-face {
+    font-family: 'Helvetica Neue';
+    src: local('Helvetica Neue'),
+        url('../../fonts/HelveticaNeueCyr-Medium.woff2') format('woff2'),
```

Comment:
Let's not carry external fonts and rely only on system ones.

Reply:
Ok, as you wish :)